postgres_entrypoint/init.sql

```diff
@@ -9,6 +9,7 @@ CREATE TABLE IF NOT EXISTS restaurant_info (
     price_range VARCHAR(10),
     vat_code INT
 );
+ALTER TABLE restaurant_info REPLICA IDENTITY FULL;
```

Comment:
What are whose changes 😮

Reply:
It's for debezium:
https://stackoverflow.com/questions/59799503/postgres-debezium-does-not-publish-the-previous-state-of-a-record